app/jinja_filters.py

```diff
@@ -563,10 +557,11 @@ def __init__(
         no_answer_provided: str,
         edit_link_text: str,
         edit_link_aria_label: str,
+        use_answer_label: bool = False,
     ) -> None:
-        self.rowTitle = strip_tags(question["title"])
         self.id = question["id"]
         self.rowItems = []
+        self.rowTitle = strip_tags(question["title"])
 
         multiple_answers = len(question["answers"]) > 1
```

Comment:
This variable is only used to determine `use_answer_label` so this could just be `use_answer_label = use_answer_label or len(question["answers"]) > 1` and then drop `multiple_answers`.

Reply:
Updated 👍